files/zh-cn/web/css/position/index.md

```diff
@@ -4,16 +4,16 @@ slug: Web/CSS/position
 ---
 {{CSSRef}}
 
-CSS **`position`**属性用于指定一个元素在文档中的定位方式。{{Cssxref("top")}}，{{Cssxref("right")}}，{{Cssxref("bottom")}} 和 {{Cssxref("left")}} 属性则决定了该元素的最终位置。
+CSS **`position`** 属性用于指定一个元素在文档中的定位方式。{{Cssxref("top")}}，{{Cssxref("right")}}，{{Cssxref("bottom")}} 和 {{Cssxref("left")}} 属性则决定了该元素的最终位置。
 
 {{EmbedInteractiveExample("pages/css/position.html")}}
 
 ### 定位类型
 
-- **定位元素（positioned element）**是其[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `relative`, `absolute`, `fixed `或 `sticky` 的一个元素（换句话说，除`static`以外的任何东西）。
-- **相对定位元素**（**relatively positioned element**）是[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `relative `的元素。
-- **绝对定位元素（absolutely positioned element）**是[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `absolute` 或 `fixed` 的元素。
-- **粘性定位元素**（**stickily positioned element**）是[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `sticky` 的元素。
+- **定位元素（positioned element）** 是其[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `relative`, `absolute`, `fixed `或 `sticky` 的一个元素（换句话说，除`static`以外的任何东西）。
+- **相对定位元素（relatively positioned element）** 是[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `relative `的元素。
+- **绝对定位元素（absolutely positioned element）** 是[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `absolute` 或 `fixed` 的元素。
+- **粘性定位元素（stickily positioned element）** 是[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `sticky` 的元素。
```

Comment:
Hi @btea, we just won't add space between symbol and chinese words. So if you want to update this section, could just follow (or do nothing on it):

```suggestion
- **定位元素**（positioned element）是其[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `relative`, `absolute`, `fixed `或 `sticky` 的一个元素（换句话说，除`static`以外的任何东西）。
- **相对定位元素**（relatively positioned element）是[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `relative `的元素。
- **绝对定位元素**（absolutely positioned element）是[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `absolute` 或 `fixed` 的元素。
- **粘性定位元素**（stickily positioned element）是[计算后](/zh-CN/docs/Web/CSS/computed_value)位置属性为 `sticky` 的元素。
```

Reply:
Ok, I see, I'll update it.